code/datums/radiation_wave.dm

```diff
@@ -68,7 +68,8 @@
 		place = cmaster_turf
 		for(var/i in 1 to distance)
 			place = get_step(place, dir)
-			atoms += get_rad_contents(place)
+			if(place)
+				atoms += get_rad_contents(place)
```

Comment:
If you hit a null location you don't need to continue trying this direction, you can skip to the next one. Break out of this loop instead
```
if(!place)
  break
```